ambry-cloud/src/main/java/com.github.ambry.cloud/azure/AzureMetrics.java

```diff
@@ -63,7 +65,8 @@
   public final Timer blobDeletionTime;
   public final Counter blobDeleteErrorCount;
   public final Counter configErrorCount;
-  public final Meter blobUploadRate;
+  public final Meter blobUploadSuccessByteRate;
+  public final Histogram blobUploadSuccessLatency;
```

Comment:
Why histogram and not timer?  Should the others also be histograms?  We are able to get mean/95/99 values of the timer metrics.

Reply:
Histogram is a combination of Meter and Timer. On ingraph, Histogram will show mean/95/99 and rate. 

Timer is OK. Timer can show mean/95/99 .

Reply:
Thanks for @cgtz  pointing out. It's timer that has histogram and meter.

Reply:
Timer used.
Renamed some metric for readability on ingrpah. 
Fixed a bug.